astroquery/esa/xmm_newton/tests/test_xmm_newton.py

```diff
@@ -73,15 +86,20 @@ def test_query_xsa_tap(self):
         xsa = XMMNewtonClass(self.get_dummy_tap_handler())
         xsa.query_xsa_tap(**parameters)
         self.get_dummy_tap_handler().check_call("launch_job", parameters)
+        self.get_dummy_tap_handler().check_parameters(parameters, "launch_job")
+        self.get_dummy_tap_handler().check_method("launch_job")
+        self.get_dummy_tap_handler().get_tables()
+        self.get_dummy_tap_handler().get_columns()
+        self.get_dummy_tap_handler().load_tables()
 
     def test_get_tables(self):
         parameters2 = {'only_names': True,
                        'verbose': True}
 
-        dummyTapHandler = DummyXMMNewtonTapHandler("get_tables", parameters2)
-        xsa = XMMNewtonClass(self.get_dummy_tap_handler())
-        xsa.get_tables(only_names=True, verbose=True)
-        dummyTapHandler.check_call("get_tables", parameters2)
+        dummyHandler = DummyXMMNewtonTapHandler("get_tables", parameters2)
+        dummyHandler.check_call("get_tables", parameters2)
```

Comment:
The previous approach was more preferred as it calls the user facing API. As I see (I may be mistaken), here there isn't an  instance of `XMMNewtonClass` called. This infact also the case in a few other tests.

Reply:
Done, thanks